crates/swc/tests/fixture/globals/env-map-computed/input/.swcrc

```diff
@@ -2,7 +2,6 @@
     "jsc": {
         "transform": {
             "optimizer": {
-                "simplify": false,
```

Comment:
Why did you touch this? Please revert

Reply:
@kdy1 Thank you for your reviews. You mean I need to modify `simplify` option which receive boolean value and json? Like this.

```rust
pub struct OptimizerConfig {
    #[serde(default)]
    pub simplify: SimplifyOption,
}

#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(untagged)]
pub enum SimplifyOption {
    Bool(bool),
    Json(SimplifyJsonOption),
}

impl Default for SimplifyOption {
    fn default() -> Self {
        SimplifyOption::Bool(true)
    }
}

#[derive(Debug, Default, Clone, Copy, Serialize, Deserialize)]
#[serde(deny_unknown_fields, rename_all = "camelCase")]
pub struct SimplifyJsonOption {
    pub preserve_imports_with_side_effects: bool,
}
```

If you think so, I will refactor in this way

Reply:
Yes, thank you!

Reply:
I updated!